tools/packaging/common/envoy_bootstrap.json

```diff
@@ -689,6 +689,17 @@
           }
         ],
         {{- end}}
+        {{- if .envoy_status_port_enable_proxy_protocol }}
+        "listener_filters": [
+          {
+            "name": "envoy.listener.proxy_protocol",
+            "typed_config": {
+              "@type": "type.googleapis.com/envoy.extensions.filters.listener.proxy_protocol.v3.ProxyProtocol",
+              "allow_requests_without_proxy_protocol": true,
```

Comment:
is this still needed now?

Reply:
The `"allow_requests_without_proxy_protocol": true` part you mean? I think so, because the Kubernetes probes won’t use proxy protocol I guess.

Reply:
I was wondering if we can have multiple filter chain matches that distinguishes health check call from AWS NLB vs K8s and this filter accoridingly

Reply:
I don't think it's possible, since the proxy protocol filter is a listener filter, or is it? Do you prefer that we add another dedicated status listener with proxy protocol support, for example on port 15022?

Reply:
I think how its written is good we can just make this opt-in

Reply:
Oh which you already did. LGTM then personally

Reply:
Oh. Sorry. This is listener filter. You can not do what I suggested and do not think adding another port is needed. I am good with this